### PR TITLE
[ELY-2208] add tests to VaultTestCase in WF elytron tool

### PR DESCRIPTION
**Pull request by nikitagibor**
https://issues.redhat.com/browse/ELY-2208

### CODE REVIEW

**Comment by Skyllarr** [comment]
Thank you for the PR @nikitagibor . Can you please add jira issue number (in square brackets) to the commit message and the PR title.

**Comment by fjuma** [comment]
@nikitagibor Thanks for updating the PR! There are some test failures that are happening in `VaultCommandTest` with these changes.

**Comment by nikitagibor** [comment]
I have fixed one test, it should not fail anymore, I also fixed couple of typos.

There are failures in Arabic, Chinese and Japanese chars tests on windows, I've tried to execute them on my local machine and ended up with same results as in CI. I investigated those failures and here is what I have found:

- forcing utf-8 encoding at compile time does not fix this issue
- if I execute vault.bat script on latest stable wildfly distribution on windows with, for example, Japanese chars, I see question marks instead of Japanese chars in console output ( tested in powershell with SimSun-ExtB font )

`vault.bat -e vault-v1/vault_new/ -k vault-v1/vault-jceks.keystore -v test -p secretsecret -i 34 -s 12345678 -b 'ユーザー名' -a 'ユーザー名' -x 'ユーザー名' 

JAVA_HOME is not set. Unexpected results may occur.
Set JAVA_HOME to the directory of your local JDK to avoid this message.
=========================================================================

  JBoss Vault Tool

  JBOSS_HOME: "C:\Users\ngibor\Downloads\wildfly-24.0.1.Final"

  JAVA: "java"

  JAVA_OPTS: "  --add-exports=java.desktop/sun.awt=ALL-UNNAMED --add-exports=java.naming/com.sun.jndi.ldap=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.security=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.management/javax.management=ALL-UNNAMED --add-opens=java.naming/javax.naming=ALL-UNNAMED"

=========================================================================

WFLYSEC0047: Secured attribute value has been stored in Vault.
Please make note of the following:
********************************************
Vault Block:?????
Attribute Name:?????
Configuration should be done as follows:
VAULT::?????::?????::1
********************************************
WFLYSEC0048: Vault Configuration commands in WildFly for CLI:
********************************************
For standalone mode:
/core-service=vault:add(vault-options=[("KEYSTORE_URL" => "vault-v1/vault-jceks.keystore"),("KEYSTORE_PASSWORD" => "MASK-2hKo56F1a3jYGnJwhPmiF5"),("KEYSTORE_ALIAS" => "test"),("SALT" => "12345678"),("ITERATION_COUNT" => "34"),("ENC_FILE_DIR" => "vault-v1/vault_new/")])
********************************************
For domain mode:
/host=the_host/core-service=vault:add(vault-options=[("KEYSTORE_URL" => "vault-v1/vault-jceks.keystore"),("KEYSTORE_PASSWORD" => "MASK-2hKo56F1a3jYGnJwhPmiF5"),("KEYSTORE_ALIAS" => "test"),("SALT" => "12345678"),("ITERATION_COUNT" => "34"),("ENC_FILE_DIR" => "vault-v1/vault_new/")])
********************************************`

**Comment by Skyllarr** [comment]
@nikitagibor I would just rename `tool/src/test/resources/vault-v1/vault_arabic_chars/VAULT.dat` to `tool/src/test/resources/vault-v1/vault_data_arabic_chars/VAULT.dat` and same with others just to be consistent with the rest of the naming in that resources folder.

File: tool/src/test/java/org/wildfly/security/tool/VaultCommandTest.java [hunk 1]
```diff
@@ -229,16 +481,7 @@ public void bulkConversionMissingLocationOption() {
...
-        try {
-            executeCommandAndCheckStatus(args);
-        } catch (Exception e) {
-            // Exception is wrapped inside ELYTOOL00012
-            Assert.assertTrue(e.getCause().getCause().getMessage().contains("ELYTOOL00022"));
-            testFailed = false;
-        }
-
-        Assert.assertFalse("Test was supposed to throw exception!", testFailed);
+        executeVaultCommandWithParams(args, false, "ELYTOOL00012");
```
Comment (Skyllarr) [inline tool/src/test/java/org/wildfly/security/tool/VaultCommandTest.java:484]: @nikitagibor Shouldn't the expectedOutput be "ELYTOOL00022" instead?

Comment (nikitagibor) [reply tool/src/test/java/org/wildfly/security/tool/VaultCommandTest.java]: I fixed it

**Comment by nikitagibor** [comment]
I have reported issue, renamed vaults and enabled tests on windows.

https://issues.redhat.com/browse/ELY-2245

**Comment by Skyllarr** [comment]
> I have reported issue, renamed vaults and enabled tests on windows.
> 
> https://issues.redhat.com/browse/ELY-2245

@nikitagibor  Thanks a lot for creating an issue. I am very sorry I was not clear with the windows tests - I meant leave them disabled, so that our CI passes. But the ELY-2245 issue should mention to enable them when it is fixed.

**Comment by nikitagibor** [comment]
@Skyllarr Ah, now I see :slightly_smiling_face: I've disabled tests on windows and added issue link.

**Comment by fjuma** [comment]
@nikitagibor Looks like there are related failures here now.

**Comment by nikitagibor** [comment]
@fjuma It should be fine now, sorry for inconvenience

**Review by fjuma — APPROVED** [review]
No worries, @nikitagibor! Thanks for working on this!

**Review by Skyllarr — APPROVED** [review]

**Comment by Skyllarr** [comment]
@nikitagibor Thank you!

**Pull request merged**